tests/RuleSetTest.php

```diff
@@ -228,6 +228,43 @@ public function providerSetDefinitionNames()
         }, $setDefinitionNames);
     }
 
+    /**
+     * @param array $set
+     *
+     * @dataProvider provideSafeSets
+     */
+    public function testNoRiskyRulesInSet(array $set)
```

Comment:
combine with `testOnlyRiskyRulesInSet` and use 2nd argument that will be expected safety